doc/Tutorials/Bokeh_Backend.ipynb

```diff
@@ -7,6 +7,10 @@
     "id": "e6a399c2-931d-445e-a80a-30ea27653f27"
    },
    "source": [
+    "<div class=\"alert alert-info\" role=\"alert\">\n",
+    "    This tutorial contains a lot of bokeh plots, which may take a little while to load and render.\n",
+    "</div>\n",
+    "\n",
```

Comment:
This does help but I was also thinking of submitting an issue to suggest some sort of loading placeholder for Bokeh plots. Would this be feasible/worthwhile?